src/store/highlight/types.ts

```diff
@@ -7,6 +7,7 @@ export type HighlightState = {
 };
 
 export type SelectionItem = {
+    canCreate: boolean;
```

Comment:
Seems like if this was an optional `error` then many of the test file changes would be unnecessary to default `canCreate: true`